demo/eip/config/application.properties

```diff
@@ -1,2 +1,159 @@
+openmrs.eip.log.level=INFO
+
+# Logging level for ONLY application class loggers
+logging.level.org.openmrs.eip=${openmrs.eip.log.level}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Tomcat Configuration *************************************************************************
+#
+# Server HTTP port.
+server.port=8083
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** H2 configuration *****************************************************************************
+#
+# Whether to enable the console
+#spring.h2.console.enabled=false
+
+# Path at which the console is available, defaults to /h2-console
+# spring.h2.console.path=
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Configuration of the OpenMRS Web App *********************************************************
+#
+# OpenMRS user credentials
+openmrs.username=${OPENMRS_USER}
+openmrs.password=${OPENMRS_PASSWORD}
+
+# OpenMRS base url
+openmrs.baseUrl=${OPENMRS_URL}/openmrs
+
+# *********************** OpenMRS Datasource config ********************************************************************
+#
+# Driver class of the openMRS datasource (should not be changed in a usual use)
+spring.openmrs-datasource.driverClassName=com.mysql.jdbc.Driver
+#
+# Dialect of the openMRS datasource (should not be changed in a usual use)
+spring.openmrs-datasource.dialect=org.hibernate.dialect.MySQLDialect
+
+# Url of the openMRS datasource, you don't have to change this value as long you've set the placeholder property values
+spring.openmrs-datasource.jdbcUrl=jdbc:mysql://${openmrs.db.host}:${openmrs.db.port}/${openmrs.db.name}
+
+# User name of the openMRS datasource
+spring.openmrs-datasource.username=${OPENMRS_DB_USER}
+
+# Password of the openMRS datasource
+spring.openmrs-datasource.password=${OPENMRS_DB_PASSWORD}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Configuration of the Management database *****************************************************
+#
+# Driver class of the management datasource (should not be changed in a usual use)
+spring.mngt-datasource.driverClassName=org.h2.Driver
+
+# Dialect of the management datasource (should not be changed in a usual use)
+spring.mngt-datasource.dialect=org.hibernate.dialect.H2Dialect
+
+# Url of the management datasource
+spring.mngt-datasource.jdbcUrl=jdbc:h2:mem:${EIP_DB_NAME};DB_CLOSE_DELAY=30;LOCK_TIMEOUT=10000
+
+# User name of the management datasource
+spring.mngt-datasource.username=${EIP_DB_USER}
+
+# Password of the management datasource
+spring.mngt-datasource.password=${EIP_DB_PASSWORD}
```

Comment:
Are we using H2 database for all EIP clients?

Reply:
No, this is only for the Demo Routes client config.